apps/dashboard/src/lib/components/Course/components/NewsFeed/store.ts

```diff
@@ -1,5 +1,11 @@
+import type { Feed } from '$lib/utils/types/feed';
 import { writable } from 'svelte/store';
 
 export const isNewFeedModal = writable({
   open: false
 });
+
+export const Feeds = writable({
+  newsFeed: [] as Feed[],
+  isLoading: false
+});
```

Comment:
Please change this store to 


```js
export const newsFeed: Feed[] = writable([])
```

`isLoading` can be a local state on the page, it doesn't need to be in the store